src/Credentials.js

```diff
@@ -22,6 +22,9 @@ export default class Credentials {
     if (params.requested) {
       payload.requested = params.requested
     }
+    if (params.verified) {
```

Comment:
I need to speak up about the increasing complexity here. IMO there is no reason to separate "verified" vs "unverified". I don't see any downsides to keeping it simple for now and always returning credentials as JWTs no matter who has verified them (self or other).